migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc

```diff
@@ -15,4 +15,24 @@ You must complete the premigration checklists before you start migrating your de
 
 include::modules/ossm-control-plane-configuration-migration-requirements.adoc[leveloffset=+1]
 
-include::modules/ossm-cluster-wide-migration-methods.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/ossm-cluster-wide-migration-methods.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-a-cluster-wide-deployment-using-the-istio-injection-label.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc[leveloffset=+2]
+
+.Next steps
+
+If you are using gateways, you must migrate them before you complete the migration process.
+
+[NOTE]
```

Comment:
Is it possible to use warning for this?
This text is general so it probably makes better sense to have that at the very end or at the beginning of `Next steps`

Reply:
Done